components/constellation/constellation.rs

```diff
@@ -2710,9 +2711,15 @@ where
         }
 
         if let Some(webgl_threads) = self.webgl_threads.as_ref() {
```

Comment:
I think you can assign an `Option<IPCReceiver>` here, with the `else` block returning `None`.